sdk/keyvault/test-resources-cleanup.py

```diff
@@ -15,26 +15,21 @@
 
 if "AZURE_KEYVAULT_URL" not in os.environ:
     raise EnvironmentError("Missing a Key Vault URL")
-
-use_pwsh = os.environ.get("AZURE_TEST_USE_PWSH_AUTH", "false")
-if use_pwsh == "true":
-    credential = AzurePowerShellCredential()
-else:
-    if "KEYVAULT_TENANT_ID" not in os.environ:
-        raise EnvironmentError("Missing a tenant ID for Key Vault")
-    if "KEYVAULT_CLIENT_ID" not in os.environ:
-        raise EnvironmentError("Missing a client ID for Key Vault")
-    if "KEYVAULT_CLIENT_SECRET" not in os.environ:
-        raise EnvironmentError("Missing a client secret for Key Vault")
-
-    credential = ClientSecretCredential(
-        tenant_id=os.environ["KEYVAULT_TENANT_ID"],
-        client_id=os.environ["KEYVAULT_CLIENT_ID"],
-        client_secret=os.environ["KEYVAULT_CLIENT_SECRET"]
-    )
+if "KEYVAULT_TENANT_ID" not in os.environ:
+    raise EnvironmentError("Missing a tenant ID for Key Vault")
+if "KEYVAULT_CLIENT_ID" not in os.environ:
+    raise EnvironmentError("Missing a client ID for Key Vault")
+if "KEYVAULT_CLIENT_SECRET" not in os.environ:
+    raise EnvironmentError("Missing a client secret for Key Vault")
 
 hsm_present = "AZURE_MANAGEDHSM_URL" in os.environ
 
+credential = ClientSecretCredential(
```

Comment:
Does this still work? I thought all pipelines have transitioned to federated auth

Reply:
This was just a change from the `7.6-preview.1` PR reverting -- I'll change this back, but it won't cause issues for now because this is a manually-used script